code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -127,12 +132,35 @@ const ProgressDot = React.createClass({
       onClick = (e) => {this.dotClicked(level.url); e.preventDefault();};
     }
 
+    if (level.kind === 'named_level') {
+      let icon = <i className={`fa ${level.icon}`} style={dotStyle} />;
+      if (level.show_progress) {
+        icon = <span style={dotStyle}>&nbsp;</span>;
```

Comment:
When the span was empty (or normal whitespace) it triggered a weird issue where the text didn't line up with the circle.  I think because the line height might be zero when there's no text?